tests/functions_tests/test_basic_math.py

```diff
@@ -157,6 +157,95 @@ def test_rdiv_backward_gpu(self): self.backward_gpu(lambda x, y: y / x)
     def test_pow_backward_gpu(self):  self.backward_gpu(lambda x, y: x ** y)
     def test_rpow_backward_gpu(self): self.backward_gpu(lambda x, y: y ** x)
 
+class TestVariableConstantArrayOp(TestCase):
+    def setUp(self):
+        self.x  = numpy.random.uniform(.5, 1, (3, 2)).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+        self.value = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+
+    def check_forward(self, op, x_data, gpu):
+        value = self.value
+        if gpu:
+            value = cuda.to_gpu(value)
+        x = Variable(x_data)
+        y = op(x, value)
+        assert_allclose(op(self.x, self.value), y.data, atol=1e-6, rtol=1e-6)
+
+    def forward_cpu(self, op):
+        self.check_forward(op, self.x, False)
+
+    # 2015/6/15 Kenta OONO <oono@preferred.jp>
+    # Commented tests do not work. See issue #43
```

Comment:
Write it as a TODO comment, and do not write date information to comments (it can be extracted from the repository log).